packages/next-swc/crates/core/src/next_dynamic.rs

```diff
@@ -260,14 +261,25 @@ impl Fold for NextDynamicPatcher {
                                                 has_ssr_false = true
                                             }
                                         }
+                                        if sym == "suspense" {
+                                            if let Some(Lit::Bool(Bool {
+                                                value: true,
+                                                span: _,
+                                            })) = match &**value {
```

Comment:
`value.as_lit()` seems better than a match expression in the rhs of if let I think

Reply:
This situation will be improved when https://github.com/rust-lang/rust/issues/87121 lands